server/chat/backend/agent/orchestrator/synthesis.py

```
@@ -0,0 +1,288 @@
+
+    # Fetch findings up to and including the wave that just completed. The
+    # final summary needs full context across waves; the needs_more decision
+    # is steered by which findings are NEW (target_wave) via the prompt.
+    finding_rows = await asyncio.to_thread(_fetch_finding_rows, incident_id, user_id, target_wave)
+    pending = [row for row in finding_rows if row.get("storage_uri")]
+    raw_bodies = await asyncio.gather(
+        *(asyncio.to_thread(_download_finding, row["storage_uri"], user_id) for row in pending),
+        return_exceptions=True,
+    ) if pending else []
+    bodies: list = []
+    for row, body in zip(pending, raw_bodies):
```

Ruff (0.15.12)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)